tests/unit_test/generic_web_api/generic_web_api_config_test.py

```diff
@@ -110,3 +111,39 @@ def test_should_read_batch_size(self):
             'batchSize': 123
         })
         assert web_api_config.batch_size == 123
+
+    def test_should_read_request_builder_name_and_parameters(self):
+        web_api_config = WebApiConfig.from_dict({
+            **MINIMAL_WEB_API_CONFIG_DICT,
+            'requestBuilder': {
+                'name': 'civi',
+                'parameters': {
+                    'fieldsToReturn': ['field1', 'field2']
+                }
+            }
+        })
+        assert isinstance(
+            web_api_config.dynamic_request_builder,
+            CiviWebApiDynamicRequestBuilder
+        )
+        assert web_api_config.dynamic_request_builder.request_builder_parameters == {
+            'fieldsToReturn': ['field1', 'field2']
+        }
+
+    def test_should_read_deprecated_request_builder_name_and_parameters(self):
+        web_api_config = WebApiConfig.from_dict({
+            **MINIMAL_WEB_API_CONFIG_DICT,
+            'urlSourceType': {
```

Comment:
Didn’t we update these fields? In the previous test it looks updated.

Reply:
Yes, that is the test for backwards compatibility. We can remove that after the change is deployed and we have updated the config.